src/intents.cs

```diff
@@ -9099,5 +9099,38 @@ interface INVoiceShortcutCenter {
 	//	[Export ("deferredLocalizedIntentsStringWithFormat:fromTable:arguments:", IsVariadic = true)]
 	//	string DeferredLocalizedIntentsStringWithFormat (string format, [NullAllowed] string table, IntPtr arguments);
 	//}
+
+	[Watch (6,0), NoMac, iOS (13,0)]
+	[Native]
+	public enum INCallAudioRoute : long
+	{
+		Unknown = 0,
+		Speakerphone,
+		Bluetooth,
+	}
```

Comment:
Probably better to move the enum to where all of the other enums are placed (line 1736).